clone_all.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Utility script to clone all Quiver repos into sibling directories of the core
+# quiver repo. Adds origin (user-specific fork) and upstream remotes.
+
+set -e
+
+function _usage {
+  echo "usage: clone_all.sh github_user"
+}
+
+github_user=$1
+
+if [[ -z $github_user ]]; then
+  _usage
+  exit 1
+fi
+
+REPOS="quiver_cache \
+  quiver_check \
+  quiver_clock \
+  quiver_collection \
+  quiver_hashcode \
+  quiver_io \
+  quiver_iterables \
+  quiver_mirrors \
+  quiver_pattern \
+  quiver_strings \
+  quiver_time"
+
+cd ..
+for repo in $REPOS; do
+  if [[ -e $repo ]]; then
+    echo "INFO: $repo already exists. Skipping."
+  else
+    git clone git@github.com:$github_user/$repo.git
```

Comment:
This script assumes one already forked all the repos. Otherwise it would crash. What if we close the _upstream_ repo and then rewrite the origin URL? This way the script will always succeed. Only when the contributor has a pull request to send would they need to create a fork.